Week1/Solution 1/solution 4.py

```diff
@@ -0,0 +1,22 @@
+#4.Write a python script to enter any number, if it is integer number, then check its armstrong or not. Print appropriate message if it is not palindrom.
+
+def armstong(s):

```

Comment:
Which is the better way to do a script using user define function or doing without using user define function?

Reply:
Both ways are good to write a script

Reply:
user define function is good because we create the new script we call the funtion.

Reply:
NO the answer is wrong, By using udf as we can reuse the code again and again by just calling the function and make our repetitive work easier and also make line of coding  less.